src/services/jwt.services.js

```diff
@@ -0,0 +1,47 @@
+const jwt = require('jsonwebtoken');
+const { userDto } = require('../dto/user.dto');
+const { ApiError } = require('../exeptions/auth.error');
+
+const sign = (payload, key, expiresIn) => jwt.sign(payload, key, { expiresIn });
+
+const verify = (payload, key) => {
+  try {
+    return jwt.verify(payload, key);
+  } catch {
+    throw ApiError.unauthorized('The token is invalid');
```

Comment:
The `ApiError` constructor arguments are incorrectly ordered. Ensure the correct order is `message, status, errors`.